third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -450,28 +462,50 @@ class BlockedToMFMA : public OpRewritePattern<tt::DotOp> {
     auto oldBType = cast<RankedTensorType>(b.getType());
     auto ctx = oldAType.getContext();
 
-    ttg::AMDMfmaEncodingAttr mfmaEnc;
+    Type aElemType = oldAType.getElementType();
+    Type bElemType = oldBType.getElementType();
+    bool withScale =
+        mfmaVersion == 4 && isF8F6F4(aElemType) && isF8F6F4(bElemType);
+
+    // If mfmaVersion == 4 and both inputs are of F8F6F4 types, we will try to
+    // use the V_MFMA_*_F8F6F4 instructions since it has higher bandwidth. If we
```

Comment:
It's not higher bandwidth, which is a memory term. It's more about higher FLOPs per cycle, which is compute term.